php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -603,7 +603,7 @@ class NDB_BVL_Feedback
     *
     * @return array
     */
-    function getThreadEntries($feedbackID)
+    static function getThreadEntries($feedbackID)
```

Comment:
this change seems out of context

Reply:
It's to be able to remove "PhanStaticCallToNonStatic" from the ignored list. All the calls to it are static, so the method should be static too.

Reply:
Just confirmed it. Dismissed